jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -462,6 +473,8 @@ public JaegerSpan start() {
           startTimeNanoTicks = clock.currentNanoTicks();
           computeDurationViaNanoTicks = true;
         }
+      } else {
+        verifyStartTimeInMicroseconds();
```

Comment:
Why do we need this? Starting a new span is never expected to throw exceptions.

Reply:
well, your API specifies passing a start timestamp in microseconds. and your internal API  logs events in microseconds (whether _fake_ or not) and does timestamp arithmetic based on this value . Thus, it makes no sense to allow a non micros value when passing supplied timestamps. This fix helps minimize the buggy behavior of the interface somehow by alerting the user he's doing something completely wrong. It'll either fail all of the times or none of times, once the user fixes his code.

Reply:
we can log an error, I am still against throwing exception

Reply:
Ok, I'll change this to only log a warning.

Reply:
Fixed